src/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,17 @@
+
+export default function Hero() {
+  return (
+    <div className="flex flex-col md:flex-row gap-12 mt-12 items-center">
+      <div className="flex flex-col text-center gap-4">
+        <h1 className="flex flex-col font-irish xl:text-5xl text-4xl font-bold">
+          Welcome to <strong className="text-red">Gaminatorium</strong>
+        </h1>
+        <span className="text-xs">Experiment. Learn. Play.</span>
```

Comment:
Where "font-irish"?
Do tego font-size na figmie to 50 px. Tu jest 32. IMO troszeczkę za mało. Obrazek też IMO większy